src/pages/edit_resource/KalturaRegistration.tsx

```diff
@@ -155,7 +155,7 @@ const KalturaRegistration: FC<KalturaRegistrationProps> = ({ expanded, onChange,
                       count={Math.ceil(totalResults / itemsPrPage)}
                       page={page}
                       onChange={(_event, value) => {
-                        handlePageChange(value);
+                        handlePageChange(value).then();
```

Comment:
gir det mer lesbarhet å hekte på en .then() her for å tydeliggjøre async metode? Lintingen i intellij vil ha det ellers i koden. Samtidig er det jo vanlig praiksis å ikke tilegner return-verdier fra en funksjon til nye variable om vi ikke trenger dem.

Reply:
Tror du kan droppe .then hvis du fjerner curly-bracketsene

Reply:
de hadde jeg ikke tenkt på, nei. Men then()-spørsmålet er det samme